md/release-notes.md

```diff
@@ -7,3 +14,15 @@
  
  Only in the case you have created custom widgets using this silent library, you need to update them by adding UI Bootstrap js as an asset.
 
+
+### Full Java 8 support
+Bonita BPM 7.5.0+ is now fully Java 8 compliant.  
+Java 8 syntax and specific APIs can now be fully used in Connectors, Actor filters, Groovy Scripts, Custom REST APIs, Custom Event Handlers ...  
+As a consequence, JRE or JDK 8+ is required to run. JRE / JDK 7 is no longer supported to run Bonita BPM 7.5.0+
```

Comment:
I will avoid the term supported here (to avoid confusion with the support activity).
I would say that Bonita BPM 7.5.0+ cannot be operated on Java 1.7 anymore.

Questions:
Aren't we suppose to use a variable instead of writing Bonita BPM? (For OEM for instance)
Can we have a Troubleshooting section with the error message you get if you use Java 7 instead of Java 8? So that this page can be found by a user when searching for the error.